app/assets/stylesheets/publish_job.scss

```diff
@@ -0,0 +1,10 @@
+// Place all the styles related to the publish_job controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.tableFixHead          { overflow: auto; height: 500px; }
+.tableFixHead thead th { position: sticky; top: 0; z-index: 1; }
+
+/* Just common table stuff. Really. */
+table  { border-collapse: collapse; width: 100%; }
+th, td { padding: 8px 16px; }
+th     { background:#eee; }
```

Comment:
I think we should avoid adding styles that apply broadly. Please use a CSS class instead -- the `.tableFixHead` could be reused as well if appropriate.